core/src/test/scala/org/apache/spark/AccumulatorSuite.scala

```diff
@@ -171,7 +173,7 @@ class AccumulatorSuite extends SparkFunSuite with Matchers with LocalSparkContex
       d.foreach {
         x => acc.localValue ++= x
       }
-      acc.value should be ( (0 to maxI).toSet)
+      assert(acc.value == (0 to maxI).toSet)
```

Comment:
Yes, most code uses "==" instead of "should be". I would favor standardizing on the former. However this change doesn't even standardize it within this file. I think it's better to do it all or not at all. Removing the extra space is trivial but at least positive.